### PR TITLE
user view course

### PR DESCRIPTION
**Pull request by d2luu**

### CODE REVIEW

File: app/views/courses/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<%= provide :title, t("courses.index") %>
+<h1><%= t "user.courses_index" %></h1>
+<div class="col-md-6 col-md-offset-3">
+  <% if @courses.any? %>
+    <% @courses.each do |course| %>
+      <div>
+      <h2><%= link_to course.title, course_path(course) %></h2>
```
Comment (tuannd255) [inline app/views/courses/index.html.erb]: ident

File: app/views/courses/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<%= provide :title, t("courses.index") %>
+<h1><%= t "user.courses_index" %></h1>
+<div class="col-md-6 col-md-offset-3">
+  <% if @courses.any? %>
+    <% @courses.each do |course| %>
+    <div>
+    	<h2><%= link_to course.title, course_path(course) %></h2>
```
Comment (vietmt) [inline app/views/courses/index.html.erb]: indent

File: app/views/courses/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<%= provide :title, t("courses.index") %>
+<h1><%= t "user.courses_index" %></h1>
+<div class="col-md-6 col-md-offset-3">
+  <% if @courses.any? %>
+    <% @courses.each do |course| %>
+    <div>
```
Comment (tuannd255) [inline app/views/courses/index.html.erb]: ident

File: app/views/courses/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<%= provide :title, t("courses.index") %>
+<h1><%= t "user.courses_index" %></h1>
+<div class="col-md-6 col-md-offset-3">
+  <% if @courses.any? %>
+    <% @courses.each do |course| %>
+			<div>
```
Comment (tuannd255) [inline app/views/courses/index.html.erb]: @@@

File: app/views/courses/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        </div>
+      </div>
+    <% end %>
+  </div>
+</div>
+<div class="col-md-6" id="course_container">
+  <h2><%= @course.title %></h2>
+  <h4><%= t "courses.description" %> : <%= @course.description %></h4>
+  <h4><%= t "courses.subject" %></h4>
+    <% if @course.course_subjects.any? %>
```
Comment (tuannd255) [inline app/views/courses/show.html.erb]: ident

**Pull request merged**